.github/workflows/bumpversion.yml

```diff
@@ -9,34 +9,13 @@ jobs:
   bump-version:
     if: "!startsWith(github.event.head_commit.message, 'bump:')"
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: ['3.x']
+    name: "Bump version and create changelog with commitizen"
     steps:
-    - uses: actions/checkout@v2
-      with:
-        token: '${{ secrets.PERSONAL_ACCESS_TOKEN }}'
-        fetch-depth: 0
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python --version
-        python -m pip install -U pip poetry
-    - name: Configure repo
-      run: |
-        git config --local user.email "action@github.com"
-        git config --local user.name "GitHub Action"
-        git pull origin master --tags
-        poetry install --no-dev
-    - name: Create bump
-      run: |
-        poetry run cz bump --yes --changelog
-        git tag
-    - name: Push changes
-      uses: Woile/github-push-action@master
-      with:
-        github_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-        tags: "true"
+      - name: Check out
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Create bump and changelog
+        uses: Woile/commitizen-action@master
```

Comment:
The repo should be commitizen-tools I think

Reply:
I just update it. Thanks! I also update the documentation on [commitizen-action](https://github.com/commitizen-tools/commitizen-action)